File: src/main/kotlin/frc/team2186/robot/autonomous/DoNothing.java
package frc.team2186.robot.autonomous;

import frc.team2186.robot.lib.interfaces.AutonomousMode;

public class DoNothing extends AutonomousMode {
    public DoNothing() {
        super("Do Nothing", true);
    }

    @Override
    public void update() {
    }

    @Override
    public void init() {
    }

    @Override
    public boolean done() {
        return true;
    }
}

File: src/main/kotlin/frc/team2186/robot/autonomous/DoNothing.kt
package frc.team2186.robot.autonomous

import frc.team2186.robot.lib.common.actionRunner
import frc.team2186.robot.lib.interfaces.SequentialAutonomousMode

class DoNothing : SequentialAutonomousMode("Do Nothing", true) {
    override val actions = actionRunner {
        action {
            println("Doing nothing")
            true
        }
    }
}
